packages/validator/src/services/prepareBeaconProposer.ts

```diff
@@ -95,11 +94,7 @@ export function pollBuilderValidatorRegistration(
               const pubkeyHex = validatorStore.getPubkeyOfIndex(index);
               if (!pubkeyHex) throw Error(`Pubkey lookup failure for index=${index}`);
               const feeRecipient = validatorStore.getFeeRecipient(pubkeyHex);
-              return validatorStore.signValidatorRegistration(
-                fromHexString(pubkeyHex),
-                fromHexString(feeRecipient),
-                slot
-              );
+              return validatorStore.getValidatorRegistration(pubkeyHex, feeRecipient, slot);
```

Comment:
> the FB team requested not to create/send new registration objects unless params change.

@g11tech I still see we send every validator registration per epoch, is it expected?

Reply:
yes, they also only keep a registration for 2 epochs, so its recommended to refresh every epoch :+1: